master/pkg/model/experiment_config.go

```diff
@@ -250,6 +250,19 @@ func (r OptimizationsConfig) Validate() []error {
 	}
 }
 
+// BindMountsConfig is the configuration for bind mounts.
+type BindMountsConfig []BindMount
+
+// UnmarshalJSON implements the json.Unmarshaler interface.
+func (b *BindMountsConfig) UnmarshalJSON(data []byte) error {
+	unmarshaled := make([]BindMount, 0)
+	if err := json.Unmarshal(data, &unmarshaled); err != nil {
+		return errors.Wrapf(err, "failed to parse bind mounts")
```

Comment:
non-blocking: No need for formatting here:
```suggestion
		return errors.Wrap(err, "failed to parse bind mounts")
```